src/types/form.ts

```diff
@@ -36,6 +36,11 @@ export enum ResponseMode {
   Email = 'email',
 }
 
+export enum SubmissionLimits {
+  Unlimited = -1,
+  StandardLimit = 1000,
+}
```

Comment:
just clarified with @syan-syan again , we don't want any default standard value. can we have the default value set in mongoose to a number type that defaults to `undefined`?

Reply:
I tried to modify this to set the form object's `submissionLimit` to `undefined` when the limit is turned off, and traced the client side code. 

The `saveForm` function in settings-form.client.directive.js calls `updateSettings`, which calls `$scope.updateForm({ update: getCurrentSettings() })`, which will call `FormApi.update` with an `update` parameter of `{ submissionLimit: undefined }` when you want to toggle off the limit. This results in an empty `PUT` call to the endpoint and nothing gets updated.

I don't think this is feasible without making a new API, which will make this a lot more complicated. It is much simpler to use a canary value that is consistently defined instead.

Reply:
Resolved. Using `null` instead of `undefined` works.